Doc for GitHub.txt

```diff
@@ -1 +1,3 @@
-Hello. patch version 1.0
\ No newline at end of file
+Hello. patch version 1.0
+version 2.0
```

Comment:
are you sure that this is second version?

Reply:
yes, i am sure